lua/autorun/server/sv_pvp_movespeed.lua

```diff
@@ -44,18 +44,19 @@ local function playerIsInBuild( ply )
     return !getPlayerPvpMode( ply )
 end
 
-local function movementMultiplier(weaponCount) 
+local function movementMultiplier(weaponCount)
+    if weaponCount < 1 then return 1 end
     return math.Clamp(1 - (1.9^( weaponCount))/100, 0, 1)
 end
 
 local function setSpeed(ply, multiplier) 
     -- set speed multiplier, 0 - 1
-    ply:SetRunSpeed(math.Clamp(baseRunSpeed*multiplier,minRunSpeed, baseRunSpeed))
-    ply:SetWalkSpeed(math.Clamp(baseWalkSpeed*multiplier,minWalkSpeed, baseWalkSpeed))
+    ply:SetRunSpeed( math.Clamp( baseRunSpeed*multiplier, minRunSpeed, baseRunSpeed ) )
+    ply:SetWalkSpeed( math.Clamp( baseWalkSpeed*multiplier, minWalkSpeed, baseWalkSpeed ) )
 end
 
-local function adjustMovementSpeed(ply) 
-    if playerIsInBuild( ply ) then 
+local function adjustMovementSpeed( ply, wepNum ) 
+    if playerIsInBuild( ply ) then
         setSpeed(ply, 1) 
         return
```

Comment:
would `if playerIsInBuild( ply ) then setSpeed(ply, 1) return end` be better. 
it would make it clear setSpeed isnt returning anything

Reply:
While that would be more clear, it would then become two statements within a single-line clause, and I'm not sure how @brandonsturgeon boi feels about that.